### PR TITLE
Fix CHANGELOG.md

### PR DESCRIPTION
**Pull request by luckysori**
The changelog broke with #1242. Let's make sure we don't forget to review changes to it on new PRs or at least keep an eye on it.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,12 +6,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
-
-- Update handling of config files
-
-[Changed]: For configuration files we now use the standard location as expected on each platform (OSX, Windows, Linux).
+- All the code since the dawn of comit-rs.
 
+### Changed
+- Move config files to standard location based on platform (OSX, Windows, Linux).
 
+[Unreleased]: https://github.com/comit-network/comit-rs/compare/1625533e04119e8496b14d5e18786f150b4fce4d...HEAD
```
Comment (tcharding) [inline CHANGELOG.md]: This was under the 'Added' heading before I touched it, is it supposed to be under the 'Changed' heading now?

Comment (luckysori) [reply CHANGELOG.md]: I think the idea is that these diffs go at the bottom of the file, see https://keepachangelog.com/en/1.0.0/.

Comment (thomaseizinger) [reply CHANGELOG.md]: This is markdown syntax for link-aliases if I am not mistaken: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

Comment (luckysori) [reply CHANGELOG.md]: Oh, yeah, I didn't even notice what it was doing haha. Instead of inlining the link you use a reference.

**Pull request merged**